napari_console/qt_console.py

```diff
@@ -55,6 +58,14 @@ def str_to_rgb(arg):
             asyncio.set_event_loop_policy(WindowsSelectorEventLoopPolicy())
 
 
+def _not_napari(n: int, frame: FrameType):
+    # in-n-out is used in napari for dependency injection.
+    for pref in ["napari", "in_n_out"]:
```

Comment:
What with modules named `napari-something`?

Reply:
should we change these to 
```suggestion
    for pref in ["napari.", "in_n_out."]:
```
then?